awscli/clidriver.py

```diff
@@ -229,6 +229,7 @@ def _handle_top_level_args(self, args):
                                            format_string=LOG_FORMAT)
             self.session.set_stream_logger('awscli', logging.DEBUG,
                                            format_string=LOG_FORMAT)
+            LOG.debug("Arguments entered to CLI: %s", repr(sys.argv[1:]))
```

Comment:
What's the need for the `repr()` call?

Small request, but would you mind moving this below the "CLI version" line?  I have some log parsing stuff that checks for this as the first line.